kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateListRow.js

```diff
@@ -121,18 +119,19 @@ define([
             _updateActionButton(jobState);
         }
 
-        function selectRow(e) {
```

Comment:
combined this into the `showHideChildRow` function, since you can't trigger one without the other